src/wp-includes/block-supports/layout.php

```diff
@@ -305,32 +305,32 @@ function wp_get_layout_style( $selector, $layout, $has_block_gap_support = false
 					'declarations' => array( 'max-width' => 'none' ),
 				)
 			);
+		}
 
-			if ( isset( $block_spacing ) ) {
-				$block_spacing_values = wp_style_engine_get_styles(
-					array(
-						'spacing' => $block_spacing,
-					)
-				);
+		if ( isset( $block_spacing ) ) {
```

Comment:
Does this change deserve a `@since`?

Reply:
Hmm yeah that's a good question. Guess it's better to add one than not!

Reply:
Done.